x-pack/plugins/security_solution/public/common/components/event_details/insights/related_cases.test.tsx

```diff
@@ -123,5 +137,45 @@ describe('Related Cases', () => {
         });
       });
     });
+
+    test('Should not open the related cases accordion when isTourActive=false', async () => {
+      mockGetRelatedCases.mockReturnValue([{ id: '789', title: 'Test Case' }]);
+      await act(async () => {
+        render(
+          <TestProviders>
+            <RelatedCases eventId={eventId} />
+          </TestProviders>
+        );
+        await waitFor(() => {
+          expect(scrollToMock).not.toHaveBeenCalled();
+          expect(
+            screen.getByTestId('RelatedCases-accordion').classList.contains('euiAccordion-isOpen')
+          ).toBe(false);
+        });
+      });
```

Comment:
It does not make much sense to _waitFor_ something not to happen, it may pass on the first attempt, even if it would have failed on the second, introducing false positives.
On these kinds of negative checks, we should do the "_expect something has not happened_" only once, when we are sure it would have happened if something went wrong.
Here we can wrap the render in the `act` alone, and do the checks outside:

```suggestion
      await act(async () => {
        render(
          <TestProviders>
            <RelatedCases eventId={eventId} />
          </TestProviders>
        );
      });
      expect(scrollToMock).not.toHaveBeenCalled();
      expect(
        screen.getByTestId('RelatedCases-accordion').classList.contains('euiAccordion-isOpen')
      ).toBe(false);
```

The confirmation that this test works is the opposite test case below, we can do the same there 👇

Reply:
ok cool, will do